setup.py

```diff
@@ -2,12 +2,15 @@
 
 from distutils.extension import Extension
 from Cython.Distutils import build_ext
+# to get the proper numpy header locations (if using anaconda)
+import numpy as np
 
 ext_modules = []
 ext_modules.append(Extension(
```

Comment:
Coincidentally, I am using the bp pieces.
Looking back at the code, it seems reasonable to note the numpy needs to be
installed before executing the setup.py.

On Tue, Jul 26, 2016 at 12:06 PM, Eric Hunsberger <notifications@github.com>
wrote:

> In setup.py <https://github.com/hunse/kitti/pull/3#discussion_r72281570>:
>
> > @@ -2,12 +2,15 @@
> >
> >  from distutils.extension import Extension
> >  from Cython.Distutils import build_ext
> > +# to get the proper numpy header locations (if using anaconda)
> > +import numpy as np
> >
> >  ext_modules = []
> >  ext_modules.append(Extension(
>
> In general, I'm not sure if this extension should be part of this package
> at all. It was some extra stuff that I threw in to play around with, but
> it's not needed for the core function of the package (which is accessing
> and manipulating the Kitti dataset), and it makes it much more difficult to
> get things installed. So rather than try to get this Numpy stuff fixed, I
> think it should just be removed.
>
> Is this extension something you're using, @jasoncorso
> <https://github.com/jasoncorso>?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/hunse/kitti/pull/3/files/0c730a117cf28087a62e954662741363dd4b6014#r72281570>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AQLpJuagutsAmQTVFWqyNuP6YxnukD80ks5qZjBygaJpZM4JL2GG>
> .
>